### PR TITLE
Fix code smell in table column width for employee test

### LINKED ISSUE #6013: [UBS] Fixing code smell in TableColumnWidthForEmployeeTest.java
SonarCloud found code smell in dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java. Too many assertions in one method. Refactoring is needed
image

### PR DESCRIPTION
**Pull request by Yurii-Midianyi**
## Summary of issue

SonarCloud found code smell in dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java. Too many assertions in one method. 

## Summary of change

Added parameterized Test instead of default one and added more test coverage by creating tests for NoArgsConstructor and Builder. Also added junit-jupiter-params in the pom file for using mentioned parameterized Test.

### CODE REVIEW

**Review by Maksym-Lenets — APPROVED** [review]

File: dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java [hunk 1]
```diff
@@ -1,50 +1,97 @@
...
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
 
 class TableColumnWidthForEmployeeTest {
+    private static Employee employee;
+    private static TableColumnWidthForEmployee tableColumnWidth;
+    static int defaultWidth;
```
Comment (RomaMocherniuk) [inline dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java]: can be `private`

File: dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java [hunk 2]
```diff
@@ -1,50 +1,97 @@
...
+
+    @BeforeAll
+    public static void setup() {
```
Comment (RomaMocherniuk) [inline dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java]: I do not see any benefits of using `BeforeAll`
We can initialize all variables like this: 
```
private static Employee employee = Employee.builder().build();
private static TableColumnWidthForEmployee tableColumnWidth  = new TableColumnWidthForEmployee(employee);
private static int defaultWidth = 120;
```

File: dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java [hunk 3]
```diff
@@ -1,50 +1,97 @@
...
+            Arguments.of(defaultWidth, tableColumnWidth.getSenderPhone()),
+            Arguments.of(defaultWidth, tableColumnWidth.getTimeOfExport()),
+            Arguments.of(defaultWidth, tableColumnWidth.getTotalOrderSum()),
+            Arguments.of(defaultWidth, tableColumnWidth.getTotalPayment()),
+            Arguments.of(defaultWidth, tableColumnWidth.getViolationsAmount()));
+    }
+
+    @ParameterizedTest
+    @MethodSource("testArguments")
+    void TableColumnWidthForEmployeeWithDefaultWidthTest(int expected, int actual) {
```
Comment (RomaMocherniuk) [inline dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java]: the method's name should not start from the capital later

File: dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java [hunk 4]
```diff
@@ -1,97 +1,90 @@
...
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TableColumnWidthForEmployeeTest {
+    private static final Employee employee = Employee.builder().build();
+    private static final TableColumnWidthForEmployee tableColumnWidth = new TableColumnWidthForEmployee(employee);
```
Comment (RomaMocherniuk) [inline dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java]: please use [the Java convention for naming](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)

File: dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java [hunk 5]
```diff
@@ -2,49 +2,86 @@
 
 import greencity.entity.user.employee.Employee;
 import org.junit.jupiter.api.Test;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import java.util.stream.Stream;
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (RomaMocherniuk) [inline dao/src/test/java/greencity/entity/table/TableColumnWidthForEmployeeTest.java]: It is a bad practice to use `*` in imports
Please do not do it

**Review by RomaMocherniuk — APPROVED** [review]

**Review by ABbondar — APPROVED** [review]

**Review by ospodaryk — APPROVED** [review]

**Pull request merged**